src/zinoargus/__init__.py

```diff
@@ -226,21 +229,26 @@ def create_argus_incidents_from_new_zino_cases(argus_incidents, zino_cases):
 def close_argus_incidents_missing_from_zino(argus_incidents, zino_cases):
     for case_id in set(argus_incidents) - set(zino_cases):
         _logger.info(
-            "Zino case %s is not cached from zino, and ready to be closed in argus",
+            "Zino case %s is not cached from Zino, and ready to be closed in Argus",
             case_id,
         )
         close_argus_incident(
             argus_incidents[case_id],
-            description="This case did not exist in zino when glue service was started",
+            description="This case did not exist in Zino when glue service was started",
         )
 
 
 def synchronize_continuously(argus_incidents: IncidentMap, zino_cases: CaseMap):
     """Continuously "poll" the Zino notification channel and update Argus accordingly"""
+    global _last_incident_refresh
     while True:
-        update = _notifier.poll(timeout=1)
+        if datetime.now() > (_last_incident_refresh + INCIDENT_REFRESH_INTERVAL):
+            # Refreshes Argus incidents at least every INCIDENT_REFRESH_INTERVAL
+            refresh_argus_incidents(argus_incidents, zino_cases)
+            _last_incident_refresh = datetime.now()
+
+        update = _notifier.poll(timeout=POLL_TIMEOUT)
```

Comment:
This changes the functionality from polling every second to every 30 seconds. I don't think this is a problem, I just wanted to point it out.